test/JDBC/upgrade/14_6/schedule

```diff
@@ -252,7 +252,6 @@ format
 msdb-dbo-syspolicy_system_health_state
 dateadd_internal_df
 sys-all_columns
-sys-all_sql_modules
```

Comment:
Why did you remove this?

Reply:
This view used the catalog table babelfish_view_def for the view definition which has been modified, so we have to write separate test files in **sys-all_sql_modules-before-14_7** format. Until then, removing this test.